templates/tapestry.js

```diff
@@ -204,7 +204,14 @@ function tapestryTool(config){
         }
 
         this.dataset.nodes = this.dataset.nodes.map(node => {
-            var updatedNode = fillEmptyFields(node, { skippable: true, behaviour: "embed", completed: false, quiz: [], showInBackpack: true })
+            var updatedNode = fillEmptyFields(node, {
+                tydeType: "Regular", 
```

Comment:
Could we have it so when you don't pass the tydeType it sets it to Regular automatically? (Related to the other related comment)